README.md

```diff
@@ -21,18 +21,45 @@ For a more in-depth overview of openCryptoki, please refer to manual
 Does not work with TPM version 2.0. We plan to remove the TPM token in a future 
 openCryptoki release or version.
 
+AIX only supports the CCA and software tokens, and both are enabled in a
+default build configuration. Other tokens are unsupported and cannot be
+force-enabled, even through the `configure` utility.
+
 ## REQUIREMENTS:
 
+### Common
+Building opencryptoki needs the following utilities.
+- flex
+- bison
+- make
+- autoconf
+- automake
+- pkg-config
+- libtool
+- m4
+- openldap-devel
+- openssl-devel
+- libcap-devel (Linux-only)
+- systemd-devel (Linux-only)
+
+These libraries are usually provided by your platform's package management
+utilities. On AIX, they must be installed from the AIX Toolbox repositories.
+
+### Tokens
 - IBM ICA - requires libica library version 3.3.0 or higher for accessing ICA
 hardware crypto on IBM zSeries.
 
-- IBM CCA - requires IBM XCrypto CEX3C card (or higher) and the CEX3C host
-libraries and tools version 4.1 (or higher).
+- IBM CCA - requires a IBM Crypto CEX3C card (or higher) and the CEX3C host
+libraries and tools version 4.1 (or higher) on Linux. On AIX, it requires CCA
```

Comment:
Linux also needs "CCA host library with version 7.1 or higher", so this should read:

- IBM CCA - requires the CCA host library with version 7.1 or higher, and IBM Crypto CEX3C card (or higher) on Linux on IBM Z. On AIX, only the IBM CEX7S (4769) crypto card is supported.